server/main.lua

```diff
@@ -64,19 +63,13 @@ AddEventHandler('entityCreated', function (entity)
     then return end
 
     local type = GetEntityType(entity)
-
-    if type ~= EntityType.Ped and type ~= EntityType.Vehicle then
-        return
-    end
-
-    local vehicle = type == EntityType.Ped and GetVehiclePedIsIn(entity, false) or entity
+    local isPed = type == EntityType.Ped
+    if not (isPed or type == EntityType.Vehicle) then return end
```

Comment:
I think the old syntax here was more readable than pulling out the not.

Reply:
what do you think about
```lua
    local isPed = type == EntityType.Ped
    local isVehicle = type == EntityType.Vehicle
    if not (isPed or isVehicle) then return end
```

Reply:
I think it's less cognitive overhead to be written as `not isPed and not isVehicle`, but if you want to keep it this way it's not a big deal.